ui/client/containers/BaseProcesses.js

```diff
@@ -123,13 +81,26 @@ class BaseProcesses extends PeriodicallyReloadingComponent {
     this.setState({showLoader: shouldShowLoader == null ? true : shouldShowLoader})
   }
 
-  prepareSearchParams(search, isSubprocess) {
-    this.queries = {
-      ...this.queries,
-      isSubprocess: isSubprocess
-    }
+  prepareSearchParams(search) {
     const query = _.pick(queryString.parse(window.location.search), this.searchItems || [])
-    return Object.assign(query, search, this.queries || {});
+    return Object.assign(query, search, this.queries || {})
+  }
+
+  loadAllClashedNames() {
+    this.loadClashedNames(defaultProcessesSearchParams, (data) => HttpService.fetchProcesses(data));
+    this.loadClashedNames(defaultSubProcessesSearchParams, (data) => HttpService.fetchProcesses(data));
+    this.loadClashedNames(defaultArchivedProcessesSearchParams, (data) => HttpService.fetchProcesses(data));
+    this.loadClashedNames({}, () => HttpService.fetchCustomProcesses())
+  }
+
+  loadClashedNames = (searchParams, fetch) => {
+    const query = _.pick(queryString.parse(window.location.search), this.searchItems || [])
+    const data = Object.assign(query, searchParams)
+    fetch(data).then(response => {
```

Comment:
We don't use fetch because it doesn't  support our policy of auth headers (OAuth2 case). Please move this method to HttpService and use axios.

Reply:
Fetch is actually a method passed as param of this function

Reply:
Upps.. :) I have'nt seen it ;p